nanoFramework.Hardware.Esp32.Rmt/TransmitterChannel.cs

```diff
@@ -390,64 +331,58 @@ public void Dispose()
         #region native calls
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void NativeInit(int gpio);
+        private extern void NativeInit(int gpio);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern bool NativeGetIdleLevel();
+        private extern bool NativeGetIdleLevel();
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern bool NativeGetIsChannelIdle();
+        private extern bool NativeGetIsChannelIdle();
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void NativeSetIsChannelIdle(bool value);
+        private extern void NativeSetIsChannelIdle(bool value);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void NativeSetIdleLevel(bool value);
+        private extern void NativeSetIdleLevel(bool value);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void NativeSetCarrierMode();
+        private extern void NativeSetCarrierMode();
 
         //[MethodImpl(MethodImplOptions.InternalCall)]
-        //private static extern uint NativeTxFillItems(int channel, RmtItem[] items, ushort offset);
+        //private extern uint NativeTxFillItems(int channel, RmtItem[] items, ushort offset);
 
         //[MethodImpl(MethodImplOptions.InternalCall)]
-        //private static extern uint NativeTxStart(int channel, bool resetIndex);
+        //private extern uint NativeTxStart(int channel, bool resetIndex);
 
         //[MethodImpl(MethodImplOptions.InternalCall)]
-        //private static extern uint NativeTxStop(int channel);
+        //private extern uint NativeTxStop(int channel);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern uint NativeWriteItems(byte[] items, bool waitTxDone);
+        private extern uint NativeWriteItems(byte[] items, bool waitTxDone);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern uint NativeWaitTxDone(int waitType);
+        private extern uint NativeWaitTxDone(int waitType);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void NativeDispose();
+        private extern void NativeDispose();
 
         //[MethodImpl(MethodImplOptions.InternalCall)]
-        //private static extern int NativeGetChannelStatus(int channel);
-
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern byte NativeGetClockDivider();
+        //private extern int NativeGetChannelStatus(int channel);
 
         //[MethodImpl(MethodImplOptions.InternalCall)]
```

Comment:
Better to remove all these commented methods. Let's keep the code as clean and tidy as possible.

Reply:
These were all existing commented out calls. Didn't really look much on the transmit side to see if they were required or not. I can remove if you think they should be.

Reply:
Yes please. Let's keep the code as clean and tidy as possible.

Reply:
@josesimoes , I have reviewed the transmitter code and i can see that some of the commented out code would be useful but would rather just leave for the moment as some of it needs to move to base class. Will come back to it when i have time to do some more samples that show its use.
So if you are happy can we merge as is ?